core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -223,6 +224,7 @@ class TransactionCoordinator(brokerId: Int,
                                        partitions: collection.Set[TopicPartition],
                                        responseCallback: AddPartitionsCallback): Unit = {
     if (transactionalId == null || transactionalId.isEmpty) {
+      info(s"Returning ${Errors.INVALID_REQUEST} error code to client for $transactionalId's add partitions request")
```

Comment:
Info level seems like overkill and maybe we should remove the reference to transactionalId in the message (since it is either null or empty).